packages/backend/src/ee/services/ai/prompts/system.ts

```diff
@@ -0,0 +1,75 @@
+import { CoreMessage } from 'ai';
+
+export const getSystemPrompt = ({
+    agentName,
+    instructions,
+    date,
+    time,
+}: {
+    agentName: string;
+    instructions: string;
+    date: string;
+    time: string;
+}): CoreMessage => ({
+    role: 'system',
+    content: `You are a helpful assistant named "${agentName}" specialized in tasks related to data analytics and data exploration.
```

Comment:
Putting `agentName` here busts the cache every time, let's consider putting this on the bottom

Reply:
I'm not 100% sure if it makes any difference but maybe having 2 system prompts, first one static and second one with variables works better with cache

Reply:
I'll experiment in the follow up PR